pkg/kubelet/kubelet_test.go

```diff
@@ -2719,26 +2719,59 @@ func TestUpdateNewNodeOutOfDiskStatusWithTransitionFrequency(t *testing.T) {
 	}
 }
 
-// FIXME: Enable me..
-func testDockerRuntimeVersion(t *testing.T) {
+func TestDockerRuntimeVersion(t *testing.T) {
 	testKubelet := newTestKubelet(t)
 	kubelet := testKubelet.kubelet
 	fakeRuntime := testKubelet.fakeRuntime
 	fakeRuntime.RuntimeType = "docker"
-	fakeRuntime.VersionInfo = "1.5.0"
-	fakeRuntime.APIVersionInfo = "1.18"
+	fakeRuntime.VersionInfo = "1.10.0-rc1-fc24"
+	fakeRuntime.APIVersionInfo = "1.22"
 	kubeClient := testKubelet.fakeKubeClient
 	kubeClient.ReactionChain = testclient.NewSimpleFake(&api.NodeList{Items: []api.Node{
-		{ObjectMeta: api.ObjectMeta{Name: testKubeletHostname}},
+		{
```

Comment:
So some of these testing changes seem orthogonal to the original issue.  I would limit the scope only the version change.